tests/testthat/test-ard_continuous.R

```diff
@@ -224,3 +224,23 @@ test_that("ard_continuous() with grouped data works", {
     ard_continuous(data = ADSL, by = ARM, variables = AGE)
   )
 })
+
+test_that("ard_continuous() with dates works and displays as expected", {
+  ard_date <- ADSL |>
+    ard_continuous(
+      variables = DISONSDT,
+      statistics = ~ continuous_variable_summary_fns(c("min", "max"))
+    )
+  expect_snapshot(ard_date)
```

Comment:
Can you add an standard deviation to the summary so we have mixed-typed results?

Reply:
done